app/models/creates_application_from_draft.rb

```diff
@@ -0,0 +1,70 @@
+class CreatesApplicationFromDraft
```

Comment:
Me being picky, but could this be called `DraftToApplicationDelegator` or something to this effect? I don't think models should be named with verbs.